src/gui/previewselectdialog.cpp

```diff
@@ -135,3 +142,36 @@ void PreviewSelect::previewButtonClicked()
 
     accept();
 }
+
+void PreviewSelectDialog::saveSettings()
+{
+    // Persist modal size
```

Comment:
You inappropriately use the word "modal".
What is "modal size"?..
Please fix all affected places.

Reply:
IMO, comments explaining trivial code pieces are redundant.

Reply:
changed to dialog